### PR TITLE
Adds 5 new mining prefabs, the 4 parts of a Sunken Ship (underwater) and the Pathology Site (space)

### PR DESCRIPTION
**Pull request by trollbreeder**
[FEAT]
It'd be nice to get #12843 fixed due to the use of airless plating outside for destroyed aesthetics in the prefabs, but it's not essential to this PR.
## About the PR 
Adds 5 mining level prefabs, 4 underwater and 1 space. The prefabs are narratively interconnected with a small, self-contained story found on written paper documents and notes strewn across the floor. Players are free to explore them, piece together what happened from the fragments given, and steal the bees and plushies via telescience. (To be honest, they are kinda cute.)

### Sunken Bridge
**Oshan and Nadir, small (16x10), 30%**
image
The torn-off section of the cockpit and the AI core. Seems it had some final words. Other than that and a printed crew manifest, not much else can be found here.
### Sunken Medbay
**Oshan and Nadir, small (12x14), 20%**
image
A small combinatory medical bay, morgue, and laundry room. Bruised miners encountering this place will rejoice as it contains quite a few medicines laying around, and even a little red medibot named Dr. Fredrickson that injects salicyclic acid or menthol instead of saline for brute or burn. Seems the cremator button was torn off by that improperly-embalmed zombie.
### Sunken Engineering
**Oshan and Nadir, large (25x18), 25%**
image
The holy trinity of mechlab, engine, and cargo, infested with zombies of former crew. That engine looks bizarre, like a giant grey trumpet jury-rigged into a pressure cooker. And these bees somehow didn't die in the crash, although the quartermaster needs some therapy for all that's happened to them. (also, don't let the bartender touch the one in the mechlab!!)
### Sunken Security
**Oshan and Nadir, medium (15x13), 15%**
image
Where criminals are interrogated and the witnesses testify what they find. Quite a few got zombified during the process, and they aren't happy about it, so you'll have to fight a small horde before grabbing what you can. That is if you can even get inside without sec access on your ID...
### Pathology Site
**Space, large (24x20), 10%**
image
The old home of a single pathologist, who worked tirelessly on beneficial pathogens under a neutral flag before finding symptom 'S-Zeta' and then promptly torching the lab and boarding it all up. I wonder where they went?
## Why's this needed? 
Nadir and Oshan (but especially nadir) need more trench prefabs! There's way too few, and these fill the void nicely.
A story spread via breadcrumbs would excite players stumbling into these prefabs for the first time who get to piece together and make theories as to what happened, plus you get to peer a little into how a non-NT ship operates.
Pathology is becoming a rare and endangered species on-station, so like any other old rejected technology, it makes sense to make a little historical pathology lab you can find off-station. It also connects back to the sunken ship for **THE LORE** (if players really are that keen on exploring and knowing what *really* happened across many rounds - or they just read the code to get the full story)

## Changelog 

```changelog
(u)Trobby
(*)Added 5 new prefabs, 4 in the trench and 1 in space!
(+)A thought-to-be-lost spaceship has recently been found torn apart and spread all over the trenches of Abzu and Magus. Maybe you can piece together what happened?
(+)We've also found a small, abandoned pathology lab out in the mining level. It seems quite dangerous...
```

### CODE REVIEW

File: code/area.dm [hunk 1]
```diff
@@ -1556,6 +1556,27 @@ area/prefab/torpedo_deposit
...
+		icon_state = "red"
+	engineering
+		name = "Sunken Engineering"
+		icon_state = "yellow"
+	bridge
+		name = "Sunken Bridge"
+		icon_state = "blue"
+
+area/prefab/pathologysite
+	name = "Derelcit Pathology Site"
```
Comment (flappybatpal) [inline code/area.dm:1577]:
```suggestion
	name = "Derelict Pathology Site"
```

**Review by flappybatpal — CHANGES_REQUESTED** [review]

**Comment by NightmareChamillian** [comment]
these are pretty impressive; well done!

**Comment by nefarious6th** [comment]
super excited for these and everything looks good imo

**Comment by pali6** [comment]
Shame that this got closed.

**Pull request closed without merging**